### ISSUE TITLE
Selected text is not very readable [BUG]

### ISSUE THREAD

**Issue by alexanderadam** [Sinacs/Joplin.Plugin.ReMoods.Theme#26]
**Describe the bug**

![grafik](https://user-images.githubusercontent.com/372620/217647036-1488c449-95ed-486d-9da8-730f63158437.png)

**To Reproduce**
1. Select text

**Expected behavior**

Text should be readable

**Comment by Sinacs** [comment]
The select text problem seems different for everyone. I can't recreate the problem so I'm not so sure why it happened. What Joplin version and OS system are you using?

**Comment by alexanderadam** [comment]
The default is different for sure but it should be able to configure it with [`::selection` selector](https://www.w3schools.com/cssref/sel_selection.php).

**Comment by Sinacs** [comment]
I will release the updates tomorrow and I hope it will work for you.

**Comment by alexanderadam** [comment]
It didn't seem to have worked 🫣

**Comment by Sinacs** [comment]
Due to my side didn't have this problem, I need your help. I need to know what has overwritten the selected text styles.

Can you help me find out the exact problem in the DOM and provide me a screenshot?

**Comment by alexanderadam** [comment]
![grafik](https://user-images.githubusercontent.com/372620/218072953-c43ff115-1710-4d59-8e1c-73e5b05a8c63.png)

I found the issue: the CodeMirror is creating additional `spans` with other definitions. Hence the native definition doesn't work :roll_eyes:

**Comment by Sinacs** [comment]
I found something weird, these few lines of code seem doesn't work for all users: [chrome.css line 689 ~ 704](https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme/blob/fc77f898286ff6e09f435be94969c0f58369d529/src/chrome.css#L689-L704) It would be perfect if you can help find out the problem within these few lines of code.

The CSS class `.CodeMirror-selectedtext` is not the perfect selector for adding background color, I can show you the difference:

If I use your solution to fix it:
```
#react-root .CodeMirror-selectedtext {
  color: var(--me-selected-text) !important;
  background: hsl(calc(var(--usp-hue) + 180deg),  10%,  45%,  50%) !important;
}
```
The output:
![Screenshot 2023-02-11 001704](https://user-images.githubusercontent.com/110529913/218142163-2c4fa092-692a-4e5e-94ca-fb44d54834c3.png)

But the original selected text style should be like this:
![Screenshot 2023-02-10 235205](https://user-images.githubusercontent.com/110529913/218142510-d27f5a0e-1c5b-4c20-b48e-11116ecbb061.png)

Overall, if I use your solution, you will lose the original selected text style and the focused state background color.

**Comment by alexanderadam** [comment]
Yes, I'm wondering why this library reinvented selected text.
What if the background of `.CodeMirror-selectedtext` is set to `none` and we're just using the proper system selection?

**Comment by Sinacs** [comment]
>What if the background of `.CodeMirror-selectedtext` is set to `none` and we're just using the proper system selection?

Exactly the same as this one:
https://user-images.githubusercontent.com/110529913/218142510-d27f5a0e-1c5b-4c20-b48e-11116ecbb061.png

ReMoods originally didn't set the background color to `.CodeMirror-selectedtext`, but there is another class `.CodeMirror-selected` and it has the same output as the `::selection` selectors. I'm just curious about why both `.CodeMirror-selected` and `::selection` didn't work for some users.

**Comment by Sinacs** [comment]
I think the main problem is within the `.CodeMirror-selected`, the purpose should be as an alternative for `::selection`. This CSS class will pop up in the DOM when we select the text in the Markdown Editor. And the location is `.CodeMirror-lines > div > div > div.CodeMirror-selected`:

![Screenshot 2023-02-13 162706](https://user-images.githubusercontent.com/110529913/218407542-2f2c356b-94d2-43b6-b7f4-2c27c7f7db75.png)

So, do you have the class `.CodeMirror-selected` appearing in the DOM when you selected the text in Markdown Editor? 

And based on your answer, I will pick one of the below as a solution:

1. If yes, please let me know why the color value is invalid, and I will fix the problem.
2. If not, I will implement a settings property due to this issue only appearing on some users:
  - I will use your solution to fix this, and users would enable it manually based on their actual needs.

**Comment by Sinacs** [comment]
The selected text background color issue has been fixed in the [Remoods Theme v4.1.7 updates](https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme/releases/tag/v4.1.7). Thank you again for all of your helpful information.

**Comment by alexanderadam** [comment]
I just updated and it works perfectly! :heart_eyes:
